src/libraries/System.Threading.Channels/ref/System.Threading.Channels.ChannelReader.ReadAllAsync.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I'd prefer this be named System.Threading.Channels.netstandard21.cs.  netcoreapp implements netstandard21, so it makes sense for it to be included in that build as well, and its name is more general than just being for one method.

Reply:
renamed